_data/versions.yml

```diff
@@ -1,7 +1,192 @@
 v3.0:
+- title: v3.0.1
+  note: |
+    22 December 2017
+
+    #### What's new
+    
+    ##### Breaking changes
+    
+    This release includes breaking changes to the data and resource model. [Review the documentation](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/) 
+    carefully before attempting an upgrade. Some highlights include: 
+    
+    - You must upgrade to Calico v2.6.5 before you can upgrade to v3.0.1.
+    
+    - Those accessing the etcd datastore directly must complete a one-time migration.
```

Comment:
This sounds like it is for users that access the etcd datastore directly. I think this should be `Calico deployments using the etcd datastore must complete a one-time migration.` or something like that.

Reply:
OK